packages/ember-htmlbars/tests/integration/component_invocation_test.js

```diff
@@ -595,367 +66,14 @@ QUnit.test('moduleName is available on _renderNode when no layout is present', f
     }
   }));
 
-  view = EmberView.extend({
+  component = Component.extend({
     [OWNER]: owner,
     layout: compile('{{#sample-component}}Derp{{/sample-component}}', {
       moduleName: templateModuleName
     })
   }).create();
 
-  runAppend(view);
-});
-
-QUnit.test('{{component}} helper works with positional params', function() {
```

Comment:
@Serabe I assume this is deleted without replacement because it's now covered in https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js ?